schemas/v1.0/azure.yaml.json

```diff
@@ -81,9 +81,9 @@
                         "ts"
                     ]
                 },
-                "moduleName": {
+                "module": {
```

Comment:
Can we discuss how we are going to version azure.yaml file?

Reply:
I'm fine with this name change but let's do two things:

1. Let's emit a warning when we see an `azure.yaml` file with `moduleName` in it saying the name is deprecated and ask the user to change.
2. Let's support using `moduleName` as an alias for `module` in our project reader for a little bit (it's fine to just land the changes to the schema now).

Let's also call this change out in the "Breaking Changes" section of the changelog (even though it doesn't technically break yet, since we will emit the warnings.

Reply:
The yaml parser we are using does not support defining multiple aliases so if we were to do this we would need some special coding.  I feel that we are too early in the project to implement this type of warning and would prefer if we just call it out in the breaking changes.

Another idea is to run a schema validation on the azure.yaml before running commands that can catch these types of things and warn the user without implementing special case scenarios.